extension/js/common/browser/browser-msg.ts

```diff
@@ -446,6 +451,40 @@ export class BrowserMsg {
     });
   }
 
+  public static thunderbirdSecureComposeHandler() {
+    const handleClickEvent = async (tabId: number, acctEmail: string, thunderbirdMsgId: number, composeMethod?: browser.compose._ComposeDetailsType) => {
+      const accountEmails = await GlobalStore.acctEmailsGet();
+      const useFullScreenSecureCompose = (await browser.windows.getCurrent()).type === 'messageCompose';
+      composeMethod = composeMethod === 'reply' || composeMethod === 'forward' ? composeMethod : undefined;
+      if (accountEmails.length !== 0) {
+        await BgUtils.openExtensionTab(
+          Url.create('/chrome/settings/inbox/inbox.htm', { acctEmail, useFullScreenSecureCompose, thunderbirdMsgId, composeMethod })
+        );
+        await browser.tabs.remove(tabId);
+      } else {
+        await BgUtils.openExtensionTab(Url.create('/chrome/settings/initial.htm', {}));
+      }
+    };
+    browser.composeAction.onClicked.addListener(async tab => {
+      const messageDetails = await browser.compose.getComposeDetails(Number(tab.id));
+      const composeMethod = messageDetails.type;
+      const msgId = Number(messageDetails.relatedMessageId);
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      const acctEmail = Str.parseEmail(messageDetails.from as string).email!;
+      await handleClickEvent(Number(tab.id), acctEmail, msgId, composeMethod);
+    });
```

Comment:
when I click `secure compose` from new message window it just opens `encrypted inbox`, but not encrypted compose modal
<img width="300" alt="Screenshot 2024-07-02 at 15 34 00" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/7182ade4-f52b-4ba8-82c3-11a55b711b91">

Reply:
Thanks @sosnovsky - can you please clarify: Does it open the email thread in FlowCrypt inbox page for that email?

For "secure compose" clicked in message display: the threadId will be loaded and would load that email.
For "secure compose" clicked in compose display - eg.via Thunderbird reply -> secure compose or Thunderbird forward -> secure compose: It will open the compose modal with quoted email rendered for forwarded or reply.

Reply:
Here I mean creating new thread by clicking `New message` button in thunderbird and then `secure compose` button in compose modal, it opens `encrypted inbox`, but without `secure compose` modal:

<img width="400" alt="Screenshot 2024-07-03 at 15 20 39" src="https://github.com/FlowCrypt/flowcrypt-browser/assets/1062093/2ba856a5-178a-4aa3-b75d-420df5edbdca">

I noticed that opened url is moz-extension://fa4f4d23-0b95-4696-a57e-f4852bf94f8e/chrome/settings/inbox/inbox.htm?acctEmail=...&useFullScreenSecureCompose=___cu_true___&thunderbirdMsgId=0, probably `thunderbirdMsgId=0` causes `secure compose` modal not appearing

Reply:
Oh ok got it - i has been corrected as of now. Tested most features and they should work fine now in this PR.